src/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html lang="en">
+  <header></header>
```

Comment:
Use emmet's html:5 shortcut for that

```suggestion
  <header>
	  <meta charset="UTF-8">
	  <meta http-equiv="X-UA-Compatible" content="IE=edge">
	  <meta name="viewport" content="width=device-width, initial-scale=1.0">
	  <title>Document</title>
  </header>
```

Reply:
исправил